docs/UsersGuide.asciidoc

```diff
@@ -1113,6 +1113,19 @@ timeouts by performing the task in background.
 This is recommended on big instances but means that the results (and
 possible errors) need to be polled via `openqa-cli api isos/$scheduled_product_id`.
 
+
+===== Statistical investigation ===== 
+In case issues appear sporadically and are therefore hard to reproduce it can help to trigger many more jobs on a production instance to gather more data first, for example the failure ratio.
+
+Example of triggering 50 jobs in the development group so that the result of passed/failed jobs is counted by openQA itself on the corresponding overview page:
```

Comment:
```suggestion
Example of triggering 50 jobs in a development group so that the result of passed/failed jobs is counted by openQA itself on the corresponding overview page:
```